docs/faq.md

```diff
@@ -2,119 +2,160 @@
+No. TinyMCE is one of the best tools for enabling rich text on the web. In Gutenberg, TinyMCE does exactly that. Nearly every text field you'll find is augmented with TinyMCE for rich text. Whether it be text, lists, or even just a single caption, TinyMCE can be invoked on blocks for rich text enhancements.
 
 ## What browsers will Gutenberg support?
 
 Gutenberg will work in modern browsers, and Internet Explorer 11.
 
 ## How do I make my own block?
-The API for creating blocks is a crucial aspect of the project. We are working on improved documentation and tutorials. Current documentation lives here: https://github.com/WordPress/gutenberg/tree/master/blocks
 
-## Does Gutenberg involve editing posts/pages in the frontend?
-No, Gutenberg is a replacement for the post and page editing screens.
+The API for creating blocks is a crucial aspect of the project. We are working on improved documentation and tutorials. Current documentation lives here
+
+## Does Gutenberg involve editing posts/pages in the front-end?
+
+No, we are designing Gutenberg primarily as a replacement for the post and page editing screens. That said, front-end editing is often confused with an editor that looks exactly like the front end. And that is something that Gutenberg will allow as themes customize individual blocks and provide those styles to the editor. Since content is designed to be distributed across so many different experiences—from desktop and mobile to full-text feeds and syndicated article platforms—we believe it's not ideal to create or design posts from just one front-end experience.
 
 ## Given Gutenberg is built in JavaScript, how will old metaboxes (PHP) work?
-We expect to have a dedicated "advanced" area at the bottom of the editor. It’s not yet implemented, we are tracking it here: https://github.com/WordPress/gutenberg/issues/952
+
+We expect to have a dedicated "advanced" area at the bottom of the editor. It’s not yet implemented, and we are tracking development here: https://github.com/WordPress/gutenberg/issues/952
 
 ## How can plugins extend the Gutenberg UI?
-The main extension point we want to emphasize is creating new blocks. We are still working on how to extend the rest of the UI that is build in JS. We are tracking it here: https://github.com/WordPress/gutenberg/issues/1352
+
+The main extension point we want to emphasize is creating new blocks. We are still working on how to extend the rest of the UI that is built in JS. We are tracking it here: https://github.com/WordPress/gutenberg/issues/1352
 
 ## Will Custom Post Types be supported?
-Yes, for the most part. If a post type disables the content field the idea is the "advanced" section at the bottom would fill the page.
+
+Yes, for the most part. If a post type disables the content field, the "advanced" section at the bottom would fill the page.
```

Comment:
Could emphasize that this is not currently the case.

Worth a mention capabilities that we might want to offer for building blocks in a custom post type?

Reply:
Yes, there are a few things you can do (like unregister a bunch of blocks for CPT) that is not the ideal interface.